relayer/naive-strategy.go

```diff
@@ -392,123 +396,86 @@ func RelayAcknowledgements(ctx context.Context, log *zap.Logger, src, dst *Chain
 		MaxMsgLength: maxMsgLength,
 	}
 
-	select {
```

Comment:
iirc we were using the `select` statement so that ctx cancellation is respected in the case of a user sending a termination signal to the running process. I think this still makes sense to keep but maybe @mark-rushakoff has more thoughts here.